src/Restorer.ts

```diff
@@ -20,15 +20,28 @@ const EVENT_NAME = "restorer:restorefocus";
 const HISOTRY_DEPTH = 10;
 
 class Restorer extends TabsterPart<RestorerProps> implements RestorerInterface {
+    private _observer: MutationObserver | undefined;
```

Comment:
This would start a mutation observer per instance of Restorer - this seems kind of scary performance wise.

We could only have one mutation observer instance on RestorerAPI, to share one mutation observer instance among all Restorer instances